proxy/envoy/resources.go

```diff
@@ -81,8 +81,16 @@ const (
 	// TCPProxyFilter is the name of the TCP Proxy network filter.
 	TCPProxyFilter = "tcp_proxy"
 
-	// MONGOProxyFilter is the name of the MONGO Proxy network filter.
-	MONGOProxyFilter = "mongo_proxy"
+	// MongoProxyFilter is the name of the Mongo Proxy network filter.
+	MongoProxyFilter = "mongo_proxy"
+
+	// RedisProxyFilter is the name of the Redis Proxy network filter.
+	RedisProxyFilter = "redis_proxy"
+
+	// RedisDefaultOpTimeout is the op timeout used for Redis Proxy filter
+	// Currently it is set to 30ms (conversion happens in the filter)
+	// TODO - Allow this to be configured.
+	RedisDefaultOpTimeout = 30 * time.Second
```

Comment:
30 milliseconds, not seconds

Reply:
intentionally set to 30s.. i.e. no timeout